habitat-lab/habitat/tasks/rearrange/sub_tasks/nav_to_obj_task.py

```diff
@@ -187,6 +192,46 @@ def reset(self, episode: Episode):
                 sim.robot.base_rot = angle_to_obj + np.pi / 2
             else:
                 sim.robot.base_rot = angle_to_obj
+        elif self._place_init:
```

Comment:
Is this file the right place to add the code for defining place inits? Nav to Obj task is the first task of OVMM while place is the last task and overall completely different from nav to obj

Reply:
Yes, right all of this will move to an ovmm_task.py. I will be pushing these changes today. I think this can be merged for now.

Reply:
Thanks Sriram